realm/src/main/java/io/realm/RealmQuery.java

```diff
@@ -172,32 +173,35 @@ private boolean containsDot(String s) {
     }
 
     /**
-     * Test if a field is null. Only works for relationships and RealmLists.
+     * Test if a field is {@code null}. Only works for not required types.
```

Comment:
"Only works nullable types."

"not required types" sounds really strange to me.